### ISSUE TITLE
CSharpier process failed to start (VSCode extension v1.7.0)

### ISSUE THREAD

**Issue by lamhm** [belav/csharpier#1238]
**Environments**

- IDE Version: `1.88.1`
- Extension Version: `1.7.0`
- CSharpier Version: `0.28.1` 
- Operating System: `Ubuntu 22.04`
- .csharpierrc Settings: `none`
- .editorconfig Settings: `none`

**Log Output**

```plaintext
["INFO" - 08:39:01] Initializing csharpier-vscode
["DEBUG" - 08:39:01] Trying to find dotnet on PATH using 'dotnet --info' 
["DEBUG" - 08:39:02] Using dotnet found from PATH at /usr/share/dotnet/dotnet
["DEBUG" - 08:39:14] /home/lamhm/.vscode-server/extensions/csharpier.csharpier-vscode-1.7.0/build
["DEBUG" - 08:39:14] /home/lamhm/.vscode-server/extensions/csharpier.csharpier-vscode-1.7.0/build
["DEBUG" - 08:39:14] Ensure there is a csharpier process for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/Controllers
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/Controllers/*.csproj
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/Controllers/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/*.csproj
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/*.csproj
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/Scripts/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/*.csproj
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/Assets/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/*.csproj
["DEBUG" - 08:39:14] Looking at /home/lamhm/Workspace/microscape-app/Assembly-CSharp-Editor.csproj
["DEBUG" - 08:39:14] Looking at /home/lamhm/Workspace/microscape-app/Assembly-CSharp.csproj
["DEBUG" - 08:39:14] Looking at /home/lamhm/Workspace/microscape-app/SimpleFileBrowser.Runtime.csproj
["ERROR" - 08:39:14] Failed while trying to read /home/lamhm/Workspace/microscape-app/SimpleFileBrowser.Runtime.csproj
["ERROR" - 08:39:14] TypeError: e.elements is not iterable
["DEBUG" - 08:39:14] Looking for /home/lamhm/Workspace/microscape-app/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Found version 0.28.1 in /home/lamhm/Workspace/microscape-app/.config/dotnet-tools.json
["DEBUG" - 08:39:14] Validating install at /home/lamhm/.cache/csharpier/0.28.1
["DEBUG" - 08:39:14] "/home/lamhm/.cache/csharpier/0.28.1/dotnet-csharpier" --version output: 0.28.1
["DEBUG" - 08:39:14] Using 0.28.1 as the version number.
["DEBUG" - 08:39:14] CSharpier at /home/lamhm/.cache/csharpier/0.28.1 already exists
["DEBUG" - 08:39:14] Adding new version 0.28.1 process for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/Controllers
["DEBUG" - 08:39:14] Warm CSharpier with initial format
["INFO" - 08:39:14] Formatting started for /home/lamhm/Workspace/microscape-app/Assets/Scripts/UI/Controllers/AbstractController.cs using CSharpier 0.28.1
["WARN" - 08:39:16] CSharpier process failed to start. Formatting cannot occur.
["WARN" - 08:39:16] CSharpier process failed to start. Formatting cannot occur.
["INFO" - 08:39:16] Formatted in 2001.5504299998283ms
["WARN" - 08:39:18] CSharpier process failed to start. Formatting cannot occur.
```

**Steps to reproduce**

- Install VSCode extension `v1.7.0`.
- Try to format any C# file using the extension (e.g. by saving the file).

**Expected behavior**

The C# file is formatted.

**Actual behavior**

Nothing happened to the C# file. The log showed that `CSharpier process failed to start`.

**Temporary workarounds**

- Downgrading the extension (to `v1.6.0`) solved the problem.
- Executing `dotnet csharpier <file_path>` in the VSCode terminal worked fine.

**Comment by belav** [comment]
I wasn't able to reproduce this but am fairly confident I know what the problem was. It is concerning that it wasn't logging any error indicating why csharpier failed to start.

Let me know if 1.7.1 resolves the problem for you.

**Comment by lamhm** [comment]
@belav: This issue is fixed with v1.7.1. Thanks a lot for your quick response.
